### PR TITLE
[Core]Minor update VariableUtils

### PR DESCRIPTION
**Pull request by loumalouomega**

### CODE REVIEW

**Review by rubenzorrilla — CHANGES_REQUESTED** [review]
Can you add a short cpp test?

File: kratos/python/add_utilities_to_python.cpp [hunk 1]
```diff
@@ -104,80 +104,73 @@ void AddUtilitiesToPython(pybind11::module& m)
...
-        .def("SumElementScalarVariable", &VariableUtils::SumElementScalarVariable<Variable<double>>)
-        .def("SumElementScalarVariable", &VariableUtils::SumElementScalarVariable<VariableComponent<VectorComponentAdaptor<array_1d<double, 3>>>>)
-        .def("SumElementVectorVariable", &VariableUtils::SumElementVectorVariable)
-        .def("AddDof", &VariableUtils::AddDof<Variable<double>>)
-        .def("AddDof", &VariableUtils::AddDof<VariableComponent<VectorComponentAdaptor<array_1d<double, 3>>>>)
-        .def("AddDof", &VariableUtils::AddDofWithReaction<Variable<double>>)
-        .def("AddDof", &VariableUtils::AddDofWithReaction<VariableComponent<VectorComponentAdaptor<array_1d<double, 3>>>>)
-        .def("CheckVariableKeys", &VariableUtils::CheckVariableKeys)
-        .def("CheckDofs", &VariableUtils::CheckDofs);
+    class_<VariableUtils > (m,"VariableUtils")
```
Comment (rubenzorrilla) [inline kratos/python/add_utilities_to_python.cpp]: Do you really need to change all these lines?

Comment (loumalouomega) [reply kratos/python/add_utilities_to_python.cpp]: It is just a tabulation correction

**Comment by loumalouomega** [comment]
Done, test added

**Comment by rubenzorrilla** [comment]
I'm ok with the tabs, but after the review please.

**Comment by loumalouomega** [comment]
@rubenzorrilla

**Comment by loumalouomega** [comment]
imagen

**Review by rubenzorrilla — APPROVED** [review]
Ok! Thank you.

**Pull request merged**